doc/ref_model/chapters/chapter08.rst

```diff
@@ -335,6 +335,12 @@ Infrastructure as well as provide:
   other Cloud Providers;
 - network orchestration solutions to manage the interconnectivity across its own and other Cloud Provider networks.
 
+The different orchestration solutions that can be found either in different techonlogy domains (e.g. Core, Edge, Radio), or different planes (Management and Control) for Service, Network and Resource layers shall consider the following attributes (not limited to the below ones):
+
+- Interoperability: Necessary integration among the orchestration components which would guarantee either the lifecycle management operations or related operation delegation to different orchestration components across the orchestration
+- Flexibiliy: There are use cases which can only be executed if the operations can run disaggregated, otherwise, extreme requirements brough y the use cases could be challenging to achieve taking in account technological constrains (e.g. use cases demanding less than 1 ms of latency)
```

Comment:
typo "brough y the use"

Reply:
It was now corrected in '[2f2e982](https://github.com/cntt-n/CNTT/pull/3254/commits/2f2e9826aabb599ea904427ae80f1557e19b4eeb)'